mlflow/R/mlflow/.travis.R

```diff
@@ -2,6 +2,8 @@ parent_dir <- dir("../", full.names = TRUE)
 package <- parent_dir[grepl("mlflow_", parent_dir)]
 install.packages(package)
 install.packages("keras", repos='http://cran.rstudio.com/')
+install.packages("roxygen2")
 library(keras)
 install_keras()
+devtools::check(error_on = "warning", args = "--no-tests")
```

Comment:
For some reason if tests are run with `devtools::check()` they're flaky so we're still going to source testthat.R like before.